ts/src/bybit.ts

```diff
@@ -1242,6 +1242,141 @@ export default class bybit extends Exchange {
         return await this.privatePostV5AccountUpgradeToUta (params);
     }
 
+    convertExpireDate (date) {
+        // parse YYMMDD to timestamp
+        const year = date.slice (0, 2);
+        const month = date.slice (2, 4);
+        const day = date.slice (4, 6);
+        const reconstructedDate = '20' + year + '-' + month + '-' + day + 'T00:00:00Z';
+        return reconstructedDate;
+    }
+
+    convertExpireDateToMarketIdDate (date) {
+        // parse 231229 to 29DEC23
+        const year = date.slice (0, 2);
+        const monthRaw = date.slice (2, 4);
+        let month = undefined;
+        const day = date.slice (4, 6);
+        if (monthRaw === '01') {
+            month = 'JAN';
+        } else if (monthRaw === '02') {
+            month = 'FEB';
+        } else if (monthRaw === '03') {
+            month = 'MAR';
+        } else if (monthRaw === '04') {
+            month = 'APR';
+        } else if (monthRaw === '05') {
+            month = 'MAY';
+        } else if (monthRaw === '06') {
+            month = 'JUN';
+        } else if (monthRaw === '07') {
+            month = 'JUL';
+        } else if (monthRaw === '08') {
+            month = 'AUG';
+        } else if (monthRaw === '09') {
+            month = 'SEP';
+        } else if (monthRaw === '10') {
+            month = 'OCT';
+        } else if (monthRaw === '11') {
+            month = 'NOV';
+        } else if (monthRaw === '12') {
+            month = 'DEC';
+        }
+        const reconstructedDate = day + month + year;
+        return reconstructedDate;
+    }
+
+    createExpiredOptionMarket (symbol) {
```

Comment:
@Dan-krm  yeah not working in the python version
```
bybit.market(BTC-28JUL23-28750-C)
Traceback (most recent call last):
  File "/Users/cjg/Git/ccxt10/ccxt/examples/py/cli.py", line 239, in <module>
    asyncio.run(main())
  File "/opt/homebrew/Cellar/python@3.10/3.10.9/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/opt/homebrew/Cellar/python@3.10/3.10.9/Frameworks/Python.framework/Versions/3.10/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/Users/cjg/Git/ccxt10/ccxt/examples/py/cli.py", line 218, in main
    result = method(*args)
  File "/Users/cjg/Git/ccxt10/ccxt/python/ccxt/async_support/bybit.py", line 1364, in market
    return self.create_expired_option_market(symbol)
  File "/Users/cjg/Git/ccxt10/ccxt/python/ccxt/async_support/bybit.py", line 1304, in create_expired_option_market
    expiry = self.yymmdd(self.safe_string(optionParts, 1))
  File "/Users/cjg/Git/ccxt10/ccxt/python/ccxt/base/exchange.py", line 1160, in yymmdd
    return Exchange.ymd(timestamp, infix, False)
  File "/Users/cjg/Git/ccxt10/ccxt/python/ccxt/base/exchange.py", line 1155, in ymd
    utc_datetime = datetime.datetime.utcfromtimestamp(int(round(timestamp / 1000)))
TypeError: unsupported operand type(s) for /: 'str' and 'int'
```

Reply:
These functions seem to expect a timestamp not a date, so I guess it is working by accident in the TS/JS side 
![image](https://github.com/ccxt/ccxt/assets/43336371/ae2f4795-c925-4cb1-b77e-f519c6165a94)